paws_collector.js

```diff
@@ -128,10 +128,19 @@ class PawsCollector extends AlAwsCollector {
     };
 
     reportDDMetric(name, value, tags = []) {
+        // check if the API key is present. This will be a good proxy for if the handler is working
+        if (!process.env.DD_API_KEY){
+            return
+        }
+
+        // Distinguish between integration and prod collectors 
+        const stackName = process.env.al_api === "api.global-integration.product.dev.alertlogic.com" ? "integration" : "cd-us-production";
```

Comment:
Collectors are deployed in different AWS accounts. I don't think we need this.

Reply:
So is there a way on the DD side to take the metrics coming form those accounts. right now the Integration metrics are showing up on the prod dashboards